yjit/src/core.rs

```diff
@@ -456,10 +461,10 @@ pub struct Context {
     reg_temps: RegTemps,
 
     /// Fields packed into u8
-    /// - Lower 7 bits: Depth of this block in the sidechain (eg: inline-cache chain)
-    /// - Top bit: Whether this code is the target of a JIT-to-JIT Ruby return
-    ///   ([Self::is_return_landing])
-    chain_depth_return_landing: u8,
+    /// - 1st bit from the left: Whether this code is the target of a JIT-to-JIT Ruby return ([Self::is_return_landing])
+    /// - 2nd bit from the left: Whether the compilation of this code has been deferred ([Self::is_deferred])
+    /// - Last 6 bits (max: 63): Depth of this block in the sidechain (eg: inline-cache chain)
+    chain_depth_return_landing_defer: u8,
```

Comment:
May want to rename this field to just flags or something like that?

Reply:
How about `chain_depth_and_flags`? https://github.com/ruby/ruby/pull/9597/commits/f2113c4df37ed69d86a6f72c4f71e3d63a93673e

Reply:
👌